src/commands/lambda/prompt.ts

```diff
@@ -20,8 +20,9 @@ import {
   SERVICE_ENV_VAR,
   VERSION_ENV_VAR,
 } from './constants'
-import {sentenceMatchesRegEx} from './functions/commons'
+import {isMissingAnyDatadogApiKeyEnvVar, isMissingDatadogSiteEnvVar, sentenceMatchesRegEx} from './functions/commons'
 
+/* tslint:disable-next-line */
```

Comment:
I'm quite not sure why that line is there, maybe when Andre added this, a `tslint:disable` was needed. It seems I could just delete it and it will work perfectly. It is worth mentioning that we just switch our linter package. Its not required anymore.